src/main/java/seedu/contax/logic/commands/ExportCsvCommand.java

```diff
@@ -60,4 +46,28 @@ public CommandResult execute(Model model) {
             return new CommandResult(MESSAGE_FAILURE);
         }
     }
+
+    /**
+     * Extracts person information into a String array for writing to CSV.
+     * @param person Person to process
+     * @return String array with person information
+     */
+    private String[] lineProcessor(Person person) {
```

Comment:
I recommend renaming it into a verb (i.e. `processLine`)

Reply:
thanks for the catch, updated in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/248/commits/2b67c04764a53ad085721a659134752196fea472